6. Developer Guides/REST API/Users/createToken.md

```diff
@@ -0,0 +1,36 @@
+---
+order: 12
+---
+
+# User Create Token
+| URL | Requires Auth | HTTP Method |
+| :--- | :--- | :--- |
+| `/api/v1/users.createToken` | `yes` | `POST` |
+
+## Payload
+| Argument | Example | Required | Description |
+| :--- | :--- | :--- | :--- |
+| `userId` or `username` | `BsNr28znDkG8aeo7W` | Required | The id or username of the user. |
+
+## Example Call
+```bash
+curl -H "X-Auth-Token: 9HqLlyZOugoStsXCUfD_0YdwnNnunAJF8V47U3QHXSq" \
+     -H "X-User-Id: aobEdbYhXfu5hkeqG" \
+     -H "Content-type:application/json" \
+     http://localhost:3000/api/v1/users.createToken \
+     -d '{ "userId": "BsNr28znDkG8aeo7W" }'
+```
+
+## Example Result
+```json
+{
+  "_id": "BsNr28znDkG8aeo7W",
+  "authToken": "2jdk99wuSjXPO201XlAks9sjDjAhSJmskAKW301mSuj9Sk",
+  "success": true
+}
+```
+
+## Change Log
+| Version | Description |
+| :--- | :--- |
+| 0.55.?? | Added |
```

Comment:
N.B. This feature hasn't been released as yet so this will need to be updated when the information is available. Happy to hold up the PR until it has been released if that's easier :)